nautobot/dcim/templates/dcim/site.html

```diff
@@ -17,11 +17,6 @@
                 <tr>
                     <td>Region</td>
                     <td>
-                        {% if object.region %}
-                            {% for region in object.region.get_ancestors %}
-                                {{ region|hyperlinked_object }} /
-                            {% endfor %}
-                        {% endif %}
```

Comment:
Removed since `region.display` now includes the ancestors of the region, so listing them individually was now redundant.